frontend/src/index.css

```
@@ -734,6 +734,61 @@
+
+.dewi-sparkle-particle {
+  position: absolute;
+  width: 0.44rem;
+  height: 0.44rem;
+  border-radius: 0;
+  clip-path: polygon(50% 0, 100% 50%, 50% 100%, 0 50%);
+  transform: translate(-50%, -50%) scale(0.35) rotate(45deg);
+  opacity: 0;
+  animation: dewi-sparkle-float 1.45s ease-out infinite;
+  animation-delay: var(--dewi-sparkle-delay, 0s);
+  box-shadow: 0 0 9px currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)